packages/integrations/mdx/README.md

```diff
@@ -249,6 +249,44 @@ const { title, fancyJsHelper } = Astro.props;
 <!-- -->
 ```
 
+### Custom components
+
+Under the hood, MDX will convert markdown into html components. For example, 
+
+```md
+> A blockquote with *some* emphasis.
+```
+
+will be converted into
+
+```html
+<blockquote>
+  <p>A blockquote with <em>some</em> emphasis.</p>
+</blockquote>
+```
+
+MDX provides a way to tap into this rendering and use your own custom components. In the above example, you could create a custom Blockquote component (in any language) that has either a `<slot />`  component or accepts a `children` prop. Then in the MDX file you import the component and export it to the `components` export. 
```

Comment:
```suggestion
But what if you want to map these blockquotes to our own markup? In the above example, you could create a custom `Blockquote` component (in any language) that has either a `<slot />`  component or accepts a `children` prop. 

[example here?]

Then, you can map the `blockquote` element to your `Blockquote` component using the `components` export:
```

Nit on explaining the "why" before diving in. Also, I think an example `Blockquote.jsx` component snippet would clarify what "slot" and "children" would be used for!